lib/usage.ts

```diff
@@ -535,10 +540,14 @@ export function usage(yargs: YargsInstance, y18n: Y18N, shim: PlatformShim) {
     (Object.keys(options.alias) || []).forEach(key => {
       options.alias[key].forEach(alias => {
         // copy descriptions.
-        if (descriptions[alias]) self.describe(key, descriptions[alias]);
+        if (descriptions[alias])
```

Comment:
@OsmanAltun we should land this work on the other PR you've created first.

I'd start working from branches by default, so that it's easier to separate out your work 👍

Reply:
I went ahead and reverted it myself, I think this branch should be ready to go, once you sign off.